captioning/utils/image_preproc.py

```diff
@@ -138,12 +137,28 @@ def get_detectron_features(image_path,
         feat_name=feat_name,
         conf_thresh=0.2
     )
+    return return_list
```

Comment:
Should we pick a more informative variable name than `return_list`?

Reply:
Changed `feat_list`